pkgs/top-level/lua-packages.nix

```diff
@@ -145,11 +146,12 @@ let
   };
 
   luasec = buildLuaPackage rec {
-    version = "0.5";
-    name = "sec-${version}";
-    src = fetchzip {
-      url = "https://github.com/brunoos/luasec/archive/luasec-${version}.tar.gz";
-      sha256 = "1zl6wwcyd4dfcw01qan7dkcw0rgzm69w819qbaddcr2ik147ccmq";
+    name = "sec-git-2015-04-17";
```

Comment:
> Or you can use pre0.6-2015-04-17

The correct ordering would be achieved by `0.6pre-currentDate`, because comparison is done component-wise lexicographically and `pre` is ordered as the very first.